### DISCUSSION TITLE
how to install and use it on mac

### DISCUSSION THREAD

**Discussion by 943m** [ORelio/Minecraft-Console-Client#1919] [Q&A]
Hey can someone please explain how to do it on mac as i am so confused. if someone could do a video tutorial it would be so appreciated

**Comment by ORelio** [comment]
1. [install Mono](https://www.mono-project.com/download/stable/#download-mac)
2. [download MinecraftClient.exe](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest)
3. run `mono MinecraftClient.exe` from a terminal
Related issues: #1064 #348 #1147
